### PR TITLE
To disable WebNotification for Crosswalk for Android

### PR DESCRIPTION
**Pull request by PeterWangIntel**
Building with "-Ddisable_notifications=1" reduces size by 68K.

The 'notifications' is an independent module in 'modules' folder.

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@d0jun

File: Source/modules/modules_gypi_generator.py [hunk 1]
```diff
@@ -140,9 +140,15 @@ def modules_idl_files():
...
       'netinfo/NetworkInformation.idl',
+    """
+
+    str_notifications = """
       'notifications/Notification.idl',
       'notifications/NotificationEvent.idl',
       'notifications/NotificationPermissionCallback.idl',
+    """
+
+    str_1 = """
```
Comment (d0ngjun) [inline Source/modules/modules_gypi_generator.py]: use a more meaningful name better.

File: Source/modules/modules_gypi_generator.py [hunk 2]
```diff
@@ -321,8 +332,14 @@ def modules_dependency_idl_files():
...
       'netinfo/NavigatorNetworkInformation.idl',
       'netinfo/WorkerNavigatorNetworkInformation.idl',
+    """
+
+    str_notifications = """
       'notifications/ServiceWorkerGlobalScopeNotifications.idl',
       'notifications/ServiceWorkerRegistrationNotifications.idl',
+    """
+
+    str_1 = """
```
Comment (d0ngjun) [inline Source/modules/modules_gypi_generator.py]: ditto

File: Source/modules/modules_gypi_generator.py [hunk 3]
```diff
@@ -382,7 +404,13 @@ def modules_event_idl_files():
       'mediastream/RTCDataChannelEvent.idl',
       'mediastream/RTCIceCandidateEvent.idl',
       'navigatorconnect/CrossOriginConnectEvent.idl',
+    """
+
+    str_notifications = """
       'notifications/NotificationEvent.idl',
+    """
+
+    str_1 = """
```
Comment (d0ngjun) [inline Source/modules/modules_gypi_generator.py]: ditto.

File: Source/modules/modules_gypi_generator.py [hunk 4]
```diff
@@ -438,9 +471,15 @@ def modules_dictionary_idl_files():
...
       'mediastream/RTCSessionDescriptionInit.idl',
+    """
+
+    str_notifications = """
       'notifications/GetNotificationOptions.idl',
       'notifications/NotificationEventInit.idl',
       'notifications/NotificationOptions.idl',
+    """
+
+    str_1 = """
```
Comment (d0ngjun) [inline Source/modules/modules_gypi_generator.py]: ditto.

File: Source/modules/modules_gypi_generator.py [hunk 5]
```diff
@@ -539,12 +583,18 @@ def generated_modules_dictionary_files():
...
+    str_notifications = """
       '<(blink_modules_output_dir)/notifications/GetNotificationOptions.cpp',
       '<(blink_modules_output_dir)/notifications/GetNotificationOptions.h',
       '<(blink_modules_output_dir)/notifications/NotificationEventInit.cpp',
       '<(blink_modules_output_dir)/notifications/NotificationEventInit.h',
       '<(blink_modules_output_dir)/notifications/NotificationOptions.cpp',
       '<(blink_modules_output_dir)/notifications/NotificationOptions.h',
+    """
+
+    str_1 = """
```
Comment (d0ngjun) [inline Source/modules/modules_gypi_generator.py]: ditto.

**Comment by halton** [comment]
@d0ngjun any update?

**Comment by d0ngjun** [comment]
LGTM.

File: Source/modules/modules_gypi_generator.py (hunk 1 above)
Comment (halton) [inline Source/modules/modules_gypi_generator.py]: I do not see this is not fixed yet.

**Comment by halton** [comment]
@PeterWangIntel I suppose you will upload rebased code and re-open the PR.

**Comment by PeterWangIntel** [comment]
The local branch corespondent to this pull is based on 'crosswalk-lite-next', but the pull target now is 'crosswalk-lite' . So, I've checkout a whole new  'crosswalk-lite' and make a clean pull request.

**Pull request closed without merging**